data/ignition/files/usr/local/bin/start-cluster-installation.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e
+
+source common.sh
+
+wait_for_assisted_service
+
+BASE_URL="http://{{.NodeZeroIP}}:8090/api/assisted-install/v2"
+
+# Get cluster id
+cluster_id=$(curl -s -S "${BASE_URL}/clusters" | jq -r .[].id)
+# Get infra_env_id
+infra_env_id=$(curl -s -S "${BASE_URL}/infra-envs"| jq -r .[].id)
+
+known_hosts=0
```

Comment:
Before the next step, shouldn't we check that the number of hosts that we expect to exist for the cluster is equal to the number of hosts that have registered with the service? This would take into account of what is specified in the AgentClusterInstall.provisionRequirements.

Reply:
Done